xlml/utils/gpu.py

```diff
@@ -379,7 +314,6 @@ def ssh_host(
   ssh_group.run(cmds, env=env)
 
 
-# TODO(piz): Check why sometime GPU instance doesn't get deleted.
 @task_group
```

Comment:
Is this `TODO(piz): Check why sometime GPU instance doesn't get deleted.` fixed?

Reply:
My previous finding was that the GPU instance was not got deleted when we manually clear the task, but it turns out TPU shows the same issue and this behavior is expected.

Reply:
We are adding those 2 PRs to clean up TPUs. Do you think you could add similar one to GPU? 

1) https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/141
2) https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/148

Reply:
Not now, but in future PRs.

Reply:
Sure, I will add GPU in the dag job.